### PR TITLE
Mvp box template experiment

### PR DESCRIPTION
**Pull request by donal-s**
Box template addition

### CODE REVIEW

File: src/data/StorageTypes.ts [hunk 1]
```diff
@@ -7,14 +7,21 @@ export type StorageAreaItemType = {
   quantity?: number;
 };
 
+export type StorageAreaBoxTemplate = {
+  boxTemplateId: string;
+  name: string;
```
Comment (byronn-1) [inline src/data/StorageTypes.ts:12]: Thanks for showing me how to make it more complicated!

File: src/App.tsx [hunk 2]
```diff
@@ -11,7 +10,7 @@ function App() {
     <Router>
       <Routes>
         <Route path='/' element={<StorageArea />}/>
-        <Route path='box/:boxId' element={<Box storageAreaContents={storageAreaContents} setBoxContents={() => {}}/>} />
+        <Route path='box/:boxId' element={<Box setBoxContents={() => {}}/>} />
```
Comment (byronn-1) [inline src/App.tsx:13]: Nice method!

Comment (donal-s) [reply src/App.tsx:13]: Done

File: src/data/StorageTypes.ts (hunk 1 above)
Comment (donal-s) [reply src/data/StorageTypes.ts:12]: :)

**Pull request merged**